index.html

```diff
@@ -0,0 +1,3179 @@
+                          <a href="#" class="subfeat_a">
+                            N.J. Salon Owner Says He's Opening With or Without Governor's Approval
+                          </a>
+                        </div>
+                      </li>
+                      <li class="sub_feat_li d_flex">
+                        <div class="ms_img_div">
+                          <a href="#" class="d_block">
+                            <picture class=" lazyside">
+                              <img
+                                src="https://d.newsweek.com/en/full/1591109/miami.jpg?w=95&h=95&l=0&t=0&f=e560f963bc35801ed3e9e9f3a6954bf1"
+                                alt="d"
+                                class="w_full"
+                              />
+                            </picture>
+                          </a>
+                        </div>
+                        <div class="info flex_1">
+                          <a href="#" class="subfeat_a">
+                            Florida Family Shaken up After SUV Crashes Through Fence, Flies Across
+                            Pool
+                          </a>
+                        </div>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+      <div class="subscribe_block">
+        <div class="row">
+          <div class="col-xl-8 col-lg-7 sub_div">
+            <div class="title_block_light subscribe_title">Subscribe</div>
+            <div class="flex_1 align-items-center sub_block d_flex">
+              <div class="s_img pos_rel fn">
+                <div class="mag_thumb prev">
+                  <img
+                    src="https://d.newsweek.com/en/full/1585087/may-08-15-2020-cover.jpg?w=170&q=95&f=c4d10af6474d34e586607b280b6589f0"
+                    alt="k"
+                    class="w_full"
+                  />
+                </div>
+                <div class="mag_thumb current">
+                  <img
+                    src="https://d.newsweek.com/en/full/1589808/may-22-2020-cover.jpg?w=170&q=95&f=96101d0d430bef83aee5a95169c9b655"
+                    alt="r"
+                    class="w_full"
+                  />
+                </div>
+              </div>
+              <div class="flex_1 s_info">
+                <h3 class="subscribe_h3 d_ib">
+                  In-depth Stories You Don't See Elsewhere About Topics You Want to Read!
+                </h3>
+                <p class="subscribe_block_p">
+                  All Access + Weekly Delivery.
+                  <br />
+                  Choose A Membership That's Perfect for You!
+                </p>
+                <button
+                  type="button"
+                  class="c_p subscribe_button subscribe_block_button d_ib pos_rel"
+                >
+                  subscribe
+                </button>
+              </div>
```

Comment:
There seems to be a small issue here with small screens

Steps to reproduce
- Open dev tools
- Click on `Toggle Device Toolbar`
- Select the small screen as shown in the screenshot

Expected Behaviour
 Should be responsive for small screens (~320px).


![image](https://user-images.githubusercontent.com/36057474/86636359-17197780-bfcc-11ea-8fad-c20b082ec4fc.png)

Reply:
Thanks for the review. I'll work on this.